try  {
    console.log("try begin");
    fun();
    console.log("try end");
}catch (error){
    console.log("error");
}